SingularityService/src/main/java/com/hubspot/singularity/data/RequestManager.java

```diff
@@ -328,8 +334,11 @@ public SingularityDeleteResult deleteRequest(SingularityRequest request, Optiona
 
     saveHistory(new SingularityRequestHistory(now, user, RequestHistoryType.DELETED, request, message));
 
+    // moves RequestState to DELETED
     SingularityDeleteResult deleteResult = delete(getRequestPath(request.getId()));
 
+    markDeleting(request, RequestHistoryType.DELETED, System.currentTimeMillis(), user, message);
+
     LOG.info("Request {} deleted ({}) by {} - {}", request.getId(), deleteResult, user, message);
```

Comment:
Yeah, might be good for the log line to suggest it was 'enqueued for delete' rather than 'deleted'